eng/pipelines/libraries/stress/ssl.yml

```diff
@@ -46,8 +46,17 @@ jobs:
 
   - bash: |
       cd '$(sslStressProject)'
+      export SSLSTRESS_CLIENT_ARGS="$SSLSTRESS_CLIENT_ARGS -http 2.0"
+      export SSLSTRESS_SERVER_ARGS="$SSLSTRESS_SERVER_ARGS -http 2.0"
       docker-compose up --abort-on-container-exit --no-color
-    displayName: Run SslStress
+    displayName: Run SslStress - HTTP/2
```

Comment:
`HTTP/2` while https://github.com/dotnet/runtime/pull/41592/files#diff-53dfbdf75e79a04b57ab654be70b8cfcR86 has `HTTP 2.0` and all 1.1 have `HTTP 1.1`.